src/syscheckd/syscheck_audit.c

```diff
@@ -671,19 +675,24 @@ void audit_parse(char *buffer) {
                     w_evt->audit_name = NULL;
                     w_evt->audit_uid = NULL;
                 } else {
-                    const char *user = get_user("",atoi(auid), NULL);
+                    char *user = get_user("",atoi(auid), NULL);
                     w_evt->audit_name = strdup(user);
```

Comment:
Please, assign the output of `get_user` function to ` w_evt->audit_name`. It is not necessary to allocate memory with `strdup`.